### DISCUSSION TITLE
Consider improvements for adapting notifications

### DISCUSSION THREAD

**Discussion by valioDOTch** [jordwest/news-feed-eradicator#386] [Ideas]
**Feature request:**
Consider trimming down on Facebook notifications, based on a more controllable set of rules, than FB offers (e.g. show only notifications from close friends, etc.)
Or as a quickfix: remov notifcations alltogether from the UI, unless the user explicitely asks for them to show them.

**Background:**
FB notifications are a major distraction for me.
I just had this experience when I looked through the FB notifications, where one notification was basically just what I describe here on Twitter
https://twitter.com/Valio_ch/status/1112983652592508928
FB tells me that I subscribed to that post, but I can't recall doing that anytime (and I deem it highly unlikely, that I ever did that).

**Comment by brandon-mork** [comment]
Proposed this low-effort alternative and submitted a pull request, to make notifications less distracting:
https://github.com/jordwest/news-feed-eradicator/issues/151
